rpcs3/Emu/SysCalls/Modules/cellMusic.cpp

```diff
@@ -99,9 +110,48 @@ s32 cellMusicSelectContents()
 	throw EXCEPTION("");
 }
 
-s32 cellMusicInitialize2()
+s32 cellMusicInitialize2(s32 mode, s32 spuPriority, vm::ptr<CellMusic2Callback> func, vm::ptr<void> userData)
 {
-	throw EXCEPTION("");
+	cellMusic.Todo("cellMusicInitialize2(mode=%d, spuPriority=%d, func=*0x%x, userData=*0x%x)", mode, spuPriority, func, userData);
+
```

Comment:
Now the quick look at prx disasm confirmed my doubts: some errors are returned immediately.